apps/namadillo/src/App/AccountOverview/StakeYourNamCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { ActionButton } from "@namada/components";
+import { routes } from "App/routes";
+import clsx from "clsx";
+import { useLocation, useNavigate } from "react-router-dom";
+
+export const StakeYourNamCard = (): JSX.Element => {
+  const navigate = useNavigate();
+  const location = useLocation();
+  return (
+    <div
+      className={clsx(
+        "flex items-center justify-between gap-12 text-sm text-cyan bg-neutral-900 rounded-sm px-6",
+        "py-6"
+      )}
+    >
+      <span className="max-w-[15ch] text-center leading-tight">
+        Stake your Transparent NAM
```

Comment:
<img width="364" alt="Screenshot 2025-04-01 at 5 43 53 PM" src="https://github.com/user-attachments/assets/20f61d6c-69ae-4811-9056-727aca35eb27" />

This button doesnt bring up the Staking route/modal. It just scrolls to the top of the page.